docker-healthcheck.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+if [[ ${healthcheck} != "true" ]]; then
+    exit 0
+fi
+
+# Set default to false for all components
+# Gets reset to true individually depending on ${service} variable
+node_check=false
+farmer_check=false
+harvester_check=false
+wallet_check=false
+
+# Determine which services to healthcheck based on ${service}
+case "${service}" in
+    all)
+        node_check=true
+        farmer_check=true
+        harvester_check=true
+        wallet_check=true
+    ;;
+    node)
+        node_check=true
+    ;;
+    harvester)
+        harvester_check=true
+    ;;
+    farmer)
+        node_check=true
+        farmer_check=true
+        harvester_check=true
+        wallet_check=true
+    ;;
+    farmer-no-wallet)
+        node_check=true
+        farmer_check=true
+        harvester_check=true
+    ;;
+    farmer-only)
+        farmer_check=true
+    ;;
+    wallet)
```

Comment:
I think with 1.3.0, this only starts the wallet, and there is no wallet-only option

Reply:
Good callout. Fixing in a commit I have staged. Will push shortly.

Reply:
Fixed. Resolving thread